src/main.py

```diff
@@ -0,0 +1,18 @@
+import psycopg2
+from flask import Flask,request,render_template
+from flask_restful import Resource,Api
+from user import *
+from community import *
+
+app = Flask(__name__)
+api = Api(app)
+
+api.add_resource(ShowComm,'/community/') # GET request to get community info.
+api.add_resource(CreateComm,'/community/') # PUT request to create a community. Method for internal use only.
```

Comment:
This should be a `POST` request, and `community` should be plural -> `communities`